init-tracing-opentelemetry/src/tracing_subscriber_ext.rs

```diff
@@ -76,7 +75,7 @@ where
     // )?;
     init_propagator()?;
     Ok(tracing_opentelemetry::layer()
-        .with_exception_field_propagation(true)
+        .with_error_records_to_exceptions(true)
```

Comment:
Why did you change it?
What is the effect?

Reply:
the other method was marked as deprecated